src/config.js

```diff
@@ -0,0 +1,12 @@
+module.exports = {
+  torusNodeEndpoints: [
+    'https://binancelabs-main-2.tor.us/jrpc',
+    'https://waseda-main-2.tor.us/jrpc',
+    'https://vgr-main-2.torusnode.com/jrpc',
+    'https://torus-main-2.torusnode.com/jrpc',
+    'https://etc-main-2.tor.us/jrpc'
+  ],
+  torusIndexes: [1, 3, 2, 5, 4],
+  supportedCurrencies: ['USD', 'AUD', 'CAD', 'EUR', 'GBP', 'HKD', 'IDR', 'JPY', 'KRW', 'RUB', 'SGD', 'UAH'],
```

Comment:
keep only the config items which are being used

Reply:
done.